pkg/webhook/webhook_cluster_role_binding.go

```diff
@@ -0,0 +1,93 @@
+//
+// Copyright (c) 2019-2020 Red Hat, Inc.
+// This program and the accompanying materials are made
+// available under the terms of the Eclipse Public License 2.0
+// which is available at https://www.eclipse.org/legal/epl-2.0/
+//
+// SPDX-License-Identifier: EPL-2.0
+//
+// Contributors:
+//   Red Hat, Inc. - initial API and implementation
+//
+
+package webhook
+
+import (
+	"context"
+
+	"github.com/devfile/devworkspace-operator/webhook/server"
+	v1 "k8s.io/api/rbac/v1"
+	apierrors "k8s.io/apimachinery/pkg/api/errors"
+	metav1 "k8s.io/apimachinery/pkg/apis/meta/v1"
+	"k8s.io/apimachinery/pkg/types"
+	crclient "sigs.k8s.io/controller-runtime/pkg/client"
+)
+
+func CreateWebhookClusterRoleBinding(client crclient.Client,
+	ctx context.Context,
+	namespace string) error {
+
+	clusterRoleBinding, err := getSpecClusterRoleBinding(namespace)
+	if err != nil {
+		return err
+	}
+
+	if err := client.Create(ctx, clusterRoleBinding); err != nil {
+		if !apierrors.IsAlreadyExists(err) {
+			return err
+		}
+		existingCfg, err := getClusterRoleBinding(ctx, client)
+		if err != nil {
+			return err
+		}
+		clusterRoleBinding.ResourceVersion = existingCfg.ResourceVersion
+		err = client.Update(ctx, clusterRoleBinding)
```

Comment:
I had an old clusterrolebinding on the cluster from an earlier test of this PR, and this step fails when the binding needs to be updated. Maybe it makes more sense to 
1. Get existing cfg
2. Compare to spec
   a. If different, delete and recreate
   b. If same, do nothing.

Reply:
I faced the same
+1 to fix before merging

Reply:
This can be done as a separate PR, IMO